python/brayns/utils/transform/transform.py

```diff
@@ -58,3 +77,29 @@ def identity(cls) -> Transform:
         :rtype: Transform
         """
         return Transform()
+
+    def apply(self, value: Vector3) -> Vector3:
+        """Apply the transform to the given value.
+
+        :param value: 3D position to transform.
+        :type value: Vector3
+        :return: Transformed value.
+        :rtype: Vector3
+        """
+        value = self.scale * value
+        value = self.rotation.apply(value)
+        return value + self.translation
+
+    def then(self, other: Transform) -> Transform:
+        """Combine transforms to be equivalent to self then other.
+
+        :param other: Transform to apply after self.
+        :type other: Transform
+        :return: Equivalent transform of self then other.
+        :rtype: Transform
+        """
+        return Transform(
+            translation=self.translation + other.translation,
```

Comment:
I this does not correct. 
`Combine transforms to be equivalent to self then other.`
The rotation of self should affect the translation of other. 

I could test the result is not the sum of the translations of chained transformations (using glm):
```
>>> tr1 = glm.rotate(glm.radians(90), glm.dvec3(0, 1, 0)) *  glm.translate(glm.dvec3(100, 0, 0))
>>> tr1
dmat4x4(( 6.12323e-17, 0, -1, 0 ), ( 0, 1, 0, 0 ), ( 1, 0, 6.12323e-17, 0 ), ( 6.12323e-15, 0, -100, 1 ))

>>> tr2 = glm.rotate(glm.radians(90), glm.dvec3(0, 0, 1)) *  glm.translate(glm.dvec3(0, 100, 0))
>>> tr2
dmat4x4(( 6.12323e-17, 1, 0, 0 ), ( -1, 6.12323e-17, 0, 0 ), ( 0, 0, 1, 0 ), ( -100, 6.12323e-15, 0, 1 ))

>>> tr = tr1 * tr2
>>> tr
dmat4x4(( 3.7494e-33, 1, -6.12323e-17, 0 ), ( -6.12323e-17, 6.12323e-17, 1, 0 ), ( 1, 0, 6.12323e-17, 0 ), ( 0, 6.12323e-15, 0, 1 ))

>>> glm.dvec3(tr[3])
dvec3( 0, 6.12323e-15, 0 )

>>> glm.dvec3(tr1[3] + tr2[3])
dvec3( -100, 6.12323e-15, -100 )
```